website/source/docs/providers/google/r/storage_bucket_object.html.markdown

```diff
@@ -8,7 +8,10 @@ description: |-
 
 # google\_storage\_bucket\_object
 
-Creates a new object inside an existing bucket in Google cloud storage service (GCS). Currently, it does not support creating custom ACLs. For more information see [the official documentation](https://cloud.google.com/storage/docs/overview) and [API](https://cloud.google.com/storage/docs/json_api).
+Creates a new object inside an existing bucket in Google cloud storage service (GCS). Currently, it does not support creating custom ACLs. For more information see 
```

Comment:
Should this link to the acl resource, which (as far as I can tell) does support custom acls?